pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -366,6 +366,16 @@ func (s *ServiceImpl) buildDashboardNavLinks(c *models.ReqContext, hasEditPerm b
 			Url:      s.cfg.AppSubURL + "/library-panels",
 			Icon:     "library-panel",
 		})
+
+		if s.features.IsEnabled(featuremgmt.FlagPublicDashboards) {
+			dashboardChildNavs = append(dashboardChildNavs, &navtree.NavLink{
+				Text:        "Public dashboards",
+				Description: "Anonymous viewing of dashboards",
```

Comment:
I don't think `Description` is a property anymore. Was getting build failures bc it doesnt exist on NavLink

Reply:
I've been wrestling with this- Description is still an actual field on NavLink. I have no idea what's going on with the linter.